### PR TITLE
E2E test setup and configuration

### PR DESCRIPTION
**Pull request by cmirandac665**

### CODE REVIEW

File: source/web/e2e/helpers.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+      //       if (error) {
+      //         return reject(error);
+      //       }
+      //       return resolve();
+      //     });
+      //   });
+      // }
+      // if (segmentServer) {
+      //   await segmentServer.stop();
+      // }
```
Comment (AlexBrandes) [inline source/web/e2e/helpers.js:145]: I think the commented out code should probably be removed here until we actually implement those features. We can always reference the original source later if we need to.

File: source/web/e2e/webdriver/index.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        if (url.match(/featureFlags$/u)) {
+          return { json: async () => clone(mockResponses.swaps.featureFlags) };
+        }
+      } else if (
+        url.match(/^https:\/\/(token-api\.airswap-prod\.codefi\.network)/u)
+      ) {
+        if (url.match(/tokens\/1337$/u)) {
+          return { json: async () => clone(mockResponses.tokenList) };
+        }
+      }
```
Comment (AlexBrandes) [inline source/web/e2e/webdriver/index.js:61]: Are these relevant to our implementation?

Comment (cmirandac665) [reply source/web/e2e/webdriver/index.js:61]: No, I will remove these.

File: source/web/e2e/helpers.js (hunk 1 above)
Comment (cmirandac665) [reply source/web/e2e/helpers.js]: Removed
